pangea-sdk/v3/service/audit/api.go

```diff
@@ -1021,3 +1028,54 @@ func (a *audit) LogStream(ctx context.Context, input pangea.ConfigIDer) (*pangea
 	var result struct{}
 	return request.DoPost(ctx, a.Client, "v1/log_stream", input, &result)
 }
+
+type ExportRequest struct {
+	pangea.BaseRequest
+
+	// Format for the records.
+	Format *DownloadFormat `json:"format,omitempty"`
+
+	// The start of the time range to perform the search on.
+	Start *time.Time `json:"start,omitempty"`
```

Comment:
In my opinion we should use higher level APIs when possible, especially when the API expects timestamps to be in a specific format which we haven't documented in the SDKs (so how is anyone supposed to know what strings will work?). Some users will be working with strings, some users will be working with `Time` objects, so there will always be someone out there that has to convert. Accepting both sounds iffy, but I could be convinced if I can see an example.

Back to this diff, I've updated these fields to `PangeaTimestamp` now; that's what that type exists for, right?

Reply:
Sorry, just checked, the problem is that dates in audit export (and the others audit api) support a timestamp and a period of time (1d, 1m, 1M) so that it should support both, dates and strings, if we don't do a struct that support both, they should be just string to allow both types. 
https://dev.pangea.cloud/docs/api/audit#/v1/export

Reply:
Updated them to strings.